### DISCUSSION TITLE
How can I centercrop an image with a given center?

### DISCUSSION THREAD

**Discussion by vectorzwt** [Project-MONAI/MONAI#7516] [Q&A] [answered]
Hi, I want to crop an image to get the roi. Firstly I get the center of my segmentation, and I want to use the CenterSpatialCropd function to center crop both my image and segmentation based on the center and roi_size, but I found that this function didn't have the roi_size parameters.  Can you give me some suggestions?

**Comment by KumoLiu** [comment, marked as answer]
Hi @vectorzwt,
The transform CenterSpatialCrop does indeed include a roi_size parameter as documentated [here](https://github.com/Project-MONAI/MONAI/blob/e9e273858b6904659b0cd581d357662dc5c782f6/monai/transforms/croppad/array.py#L507).
If you are aiming to crop around a specific center point, you might want to consider the SpatialCrop transform, as it provides greater flexibility in specifying the center point for cropping. You can find its documentation [here](https://github.com/Project-MONAI/MONAI/blob/e9e273858b6904659b0cd581d357662dc5c782f6/monai/transforms/croppad/array.py#L443).

Hope it helps, thanks.

**Reply by vectorzwt** [reply]
I got it. Thank you very much!
